src/method/process.tex

```diff
@@ -2,23 +2,123 @@
 \section{Development process}\label{sec:development-process}
 % textidote: ignore end
 
-\subsection{Development environment}\label{subsec:development}
+In this section the different elements of the development process is described.
```

Comment:
Weird sentence, should use "are" rather than "is". I'd personally rewrite it as "This section describes the different elements of the development process"

Reply:
Agreed. Will fix